src/docfx/config/ops/OpsConfigLoader.cs

```diff
@@ -12,15 +12,16 @@ namespace Microsoft.Docs.Build
 {
     internal static class OpsConfigLoader
     {
-        public static OpsConfig? LoadOpsConfig(ErrorBuilder errors, Package package, string workingDirectory = ".")
+        public static OpsConfig? LoadOpsConfig(ErrorBuilder errors, Package package, Repository? repository)
         {
-            var fullPath = new PathString(Path.Combine(workingDirectory, ".openpublishing.publish.config.json"));
+            var opDirRelativeToPackage = Path.GetRelativePath(package.BasePath, repository?.Path ?? package.BasePath);
```

Comment:
If `repository` is null, then return `null` immediately. The ops.config is currently tied to a repo.